lib/govuk_app_config/govuk_json_logging.rb

```diff
@@ -16,6 +16,12 @@ def add_custom_fields(&block)
   end
 
   def self.configure(&block)
+    # Fixes the monkey patch from the logstasher gem to support Rails 7
+    config = Rails.application.config.logstasher
+    if (!config.controller_monkey_patch && config.controller_monkey_patch != false) || config.controller_monkey_patch == true
+      require_relative "./govuk_json_logging/rails_ext/action_controller/metal/instrumentation"
+    end
+
```

Comment:
This is a copy pasta from the logstasher gem to keep the conditional require:
https://github.com/shadabahmed/logstasher/blob/0d7e3d1e79620d837accb6f111758d24c61b6b49/lib/logstasher.rb#L110

Reply:
😅  what an if condition...

is there no way to refer to this module within the logstasher gem, without vendoring it?

Reply:
ah sorry, not sure what you mean?

Reply:
I meant if the code in `instrumentation` was the same as https://github.com/shadabahmed/logstasher/blob/0d7e3d1e79620d837accb6f111758d24c61b6b49/lib/logstasher/rails_ext/action_controller/metal/instrumentation.rb, was there some way to refer to it that doesn't involve copying it into this gem. But I can see from the comments you've added that it is actually different, so no worries!